Changed name of generated documents in _generate_report.

xliiv opened the pull request:
This change adds more information to doc's title.

mkurek commented:
could you write (unit) test for it (mocking external service result)?

szok commented:
:ok:

mkurek commented on src/ralph/back_office/tests/test_models.py:
```diff
@@ -277,3 +281,17 @@ def test_return_report_when_user_not_assigned(self):
+        attachment = BackOfficeAsset._generate_report(
+            report_template.name, request, instances, report_template.language)
+
+        self.assertEqual(attachment.file.read(), GENERATED_FILE_CONTENT)
```
please validate here original name of the asset as well

Commit:
Changed name of generated documents in _generate_report.

This change generates title like:
    <date>_<lowercase-first-name>-<lowercase-lastname>_<report-name>.pdf
e.g:
    "2016-01-19_james-bond_Report.pdf"

mkurek commented:
:ok:
